src/Wallet.js

```diff
@@ -7,40 +7,34 @@ if (typeof BROWSER_RUNTIME != 'undefined' && BROWSER_RUNTIME) {
     lib = require('@ngua/cardano-serialization-lib-nodejs');
 }
 
-// _enableNami :: Effect (Promise NamiConnection)
-exports._enableNami = () => window.cardano.nami.enable();
+ALLOWED_APIS = new Set([ "nami", "gerowallet" ]);
 
-// _getNamiAddress :: NamiConnection -> Effect (Promise String)
-exports._getNamiAddress = nami => () =>
+// _enable :: String -> Effect (Promise Cip30Connection)Q
+exports._enable = api => () =>
+  ALLOWED_API.has(api) ? eval(`window.cardano.${api}.enable()`) : undefined;
```

Comment:
`eval` call is definitely not needed here.

Use `cardano[api].enable()`.

Also no need to use ternary operator:

```
exports._enable = api => () => {
  if (ALLOWED_API.has(api)) {
    window.cardano[api].enable()
  }
}
```

Reply:
Sorry for the potentially silly question but does this need to be `ALLOWED_APIS` (note plural)